### PR TITLE
Added a nbody simulation

### PR DESCRIPTION
**Pull request by RedPointyJackson**
Newtonian n-body simulation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -103,6 +103,10 @@ When in SSH:
     cd /vagrant
     python3 chaos.py
 
+## Nbody simulation status
+
+![So chaotic](data/imgs/nbody.png)
```
Comment (viktorsec) [inline README.md]: should link to chaosthebot.com

**Comment by PlasmaPower** [comment]
Add the libs you need to `requirements.txt`.

File: README.md (hunk 1 above)
Comment (PlasmaPower) [reply README.md]: And the nbody.png in git should be gitignored.

**Review by PlasmaPower — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (RedPointyJackson) [reply README.md]: @viktorsec Sorry, can you be more specific? I don't understand what should I change.

Comment (RedPointyJackson) [reply README.md]: @PlasmaPower Done! Maybe I have messed up with relative paths, though.

Comment (PlasmaPower) [reply README.md]: @RedPointyJackson currently you're getting the `nbody.png` from GitHub. That version doesn't change. You need to host the image on port 8080 or something, and then link to that.

Comment (RedPointyJackson) [reply README.md]: @PlasmaPower You are absolutely right, but I'm afraid I have no idea of how to do that. Should I close the PR?

Comment (PlasmaPower) [reply README.md]: Just move the image to server/static/imgs and the existing server should serve it.

**Pull request closed without merging**